modules/thefuck/init.zsh

```diff
@@ -0,0 +1,24 @@
+#
+# Integrates TheFuck
+#
+# Author:
+#   Luciano Mores <leslau@gmail.com>
+#
+
+if (( ! $+commands[thefuck] )); then
+    echo 'thefuck is not installed, you should "pip install thefuck" first'
+    return 1
+fi
+
+# Register alias
+eval "$(thefuck --alias)"
```

Comment:
shouldn't it be `eval $(thefuck --alias)`? see https://github.com/nvbn/thefuck

Reply:
it does the same thing